### PR TITLE
basic player functionality

### PR DESCRIPTION
**Pull request by anuejn**

### CODE REVIEW

**Comment by anuejn** [comment]
Peek 2021-09-08 02-35

**Comment by anuejn** [comment]
Tested with 
[Uploading test.audapolis…]()

File: app/src/components/Editor.tsx [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+  time: number
+): { x: number; y: number } {
+  const item = skipToTime(time, documentIterator(content)).next().value;
+  if (!item) {
+    return { x: 0, y: 0 };
+  }
+
+  const itemElement = ref.current?.children
+    .item(item.paragraphIdx)
+    ?.children.item(item.itemIdx * 2) as HTMLDivElement;
```
Comment (pajowu) [inline app/src/components/Editor.tsx:182]: This assumes every content item produces 2 html items. This is not the case for `silence`s

Comment (anuejn) [reply app/src/components/Editor.tsx:182]: yup this is broken atm but fixed in #9 . maybe I can cherry-pick the changes to this pr

**Comment by anuejn** [comment]
this should be ready to merge now :)

**Review by pajowu — APPROVED** [review]

**Pull request merged**